_data/clients/pidgin.yml

```diff
@@ -1,7 +1,4 @@
 name: Pidgin
-tracking_issue: https://developer.pidgin.im/ticket/16801
+tracking_issue: https://github.com/gkdr/lurch/issues/1
```

Comment:
I think we should keep the original ticket. It has way more information + feedback from the developers and links to all the Plugins.